controllers/snapshot/snapshot_adapter.go

```diff
@@ -435,9 +436,9 @@ func (a *Adapter) EnsureAllReleasesExist() (controller.OperationResult, error) {
 		if er != nil {
 			a.logger.Error(er, "Failed to mark snapshot integration status as invalid",
 				"snapshot.Name", a.snapshot.Name)
-			return controller.RequeueWithError(errors.Join(err, er))
+			return a.RequeueIfYoungerThanThreshold(errors.Join(err, er))
```

Comment:
It seems like this line and the other three like it are the cause of the "Duplicated Lines" error in Sonarcloud.  Is there any way to get around this? It doesn't look like I can combine the lines in any way.

Reply:
I'd just ignore it